kvbc/tools/object_store_utility/integrity_checker.cpp

```diff
@@ -216,17 +215,14 @@ void IntegrityChecker::validateRange(const BlockId& until_block) const {
 void IntegrityChecker::validateKey(const std::string& key) const {
   // Retrieve the latest block number for specified key
   Sliver sKey = Sliver::copy(key.data(), key.length());
-  auto [containing_block, _] = s3_dbset_.dbAdapter->getValue(sKey, 0);
-  UNUSED(_);
-  auto containing_block_id = util::to<BlockId>(containing_block.data());
+  auto containing_block_id = util::to<BlockId>(s3_dbset_.dbAdapter->getValue(sKey, 0).first.data());
```

Comment:
It is a valid code, but parameter unpack is more appropriate in my opinion (more "mdoren C++" and makes code more readble). To handle the "ugly" UNUSED, do:
`[[maybe_unused]] auto [containing_block, _] = s3_dbset_.dbAdapter->getValue(sKey, 0);`
same for the rest of the code.

bottom line: i'm not sure why you removed all parameter pack and took it back to c++11 style, (although its a valid code).

Reply:
gcc/g++ compilation fails for `[[maybe_unused]]`. Hence, not retaining parameter pack.